src/controllers/dynakube/oneagent/daemonset/daemonset.go

```diff
@@ -27,7 +27,7 @@ const (
 
 	hostRootMount = "host-root"
 
-	relatedImageEnvVar = "RELATED_IMAGE_DYNATRACE_ONEAGENT"
+	relatedImage = "registry.connect.redhat.com/dynatrace/oneagent:latest"
```

Comment:
related image should not be used at all anymore, so the env variable can be fully removed

Reply:
If possible we could use instead of that relatedImage field the "DEPLOYED_VIA_OLM" directly in the image() function, so we avoid setting images at two different places, and avoid passing variables via the builderInfo